src/k8s/cmd/k8s/k8s_get_join_token.go

```diff
@@ -9,10 +9,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const minTokenTTL = 3 * time.Second
```

Comment:
ok, so you can't make a token that is valid for 2 seconds but 4 seconds is fine?  Am i reading that right... no complaints just making sure i understand.

Reply:
So, I added this because we also do this for the `timeout` flag. My assumption was that this is done to avoid error if the user assumes a unit (e.g. seconds) and does not set them in the flag. However, it seems like this would just fail:
```
➜ sudo k8s get-join-token test --expires-in 10
Error: invalid argument "10" for "--expires-in" flag: time: missing unit in duration "10"
Usage:
  k8s get-join-token <node-name> [flags]

Flags:
      --expires-in duration   the time until the token expires (default 24h0m0s)
  -h, --help                  help for get-join-token
      --timeout duration      the max time to wait for the command to execute (default 1m30s)
      --worker                generate a join token for a worker node
 ```
 
So, I don't see a reason to actually have this check - removed it.